### DISCUSSION TITLE
How can I export the Queries list or get the data for it for further analysis?

### DISCUSSION THREAD

**Discussion by gareth-hughes-cloudpay** [glowroot/glowroot#951] [Q&A] [answered]
Hi,
I am very interested in exporting the Queries list for our entire application so I can refine the data. In particular I want to see all the queries to a particular db table and I think there is a goldmine of info that's in the Glowroot DB but very hard to read on the screen.
Does anyone have any ways?
thanks a lot

**Comment by gmtjlmartin** [comment, marked as answer]
I have done this by pulling the data into Excel using the chrome browser debug mode. I have attached instructions outlining the process. Note that you will get truncated SQL statements (glowroot does not store the entire SQL statement).  Also note that there is a LIMIT BUCKET which is a catch all for SQL after the max number of SQL statements that can be tracked is reached.  LIMIT BUCKET will at least give you an idea of how much SQL was not tracked and gives you a full accounting of SQL response time over the time slice you are analyzing. I hope this helps. 
 
[Instructions for Enhanced Tracking of SQL using glowroot.docx](https://github.com/glowroot/glowroot/files/9802320/Instructions.for.Enhanced.Tracking.of.SQL.using.glowroot.docx)

**Reply by gareth-hughes-cloudpay** [reply]
Hey that's great. Many thanks.
Looking at the API calls further there's a call to get the full SQL of a particular query, so it would be possible to iterate through the results and pull each full query. But that's overkill.
Thanks again for a very handy explanation

**Comment by gorkalertxundi** [comment]
Hi, this post is interesting: https://gorka.cloud/exporting-glowroot-reports-from-production-servers-to-developer-environments/
